configs/defs/centos.textproto

```diff
@@ -0,0 +1,1997 @@
+    remediation:
+      "Run the following command to install sudo."
+      "# yum install sudo"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/bin/sudo\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "  }"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/sbin/sudo\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "  }"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/usr/bin/sudo\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "  }"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/usr/sbin/sudo\"}}"
+      "      existence:{should_exist:true}"
+      "    }"
+      "  }"
+      "}"
+  }
+}
+benchmark_configs: {
+  id: "sudo-commands-use-pty"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:centos:centos:7.*" version: "3.1.2" benchmark_document: "CIS Centos Linux 7.x" }
+    title: "Ensure sudo commands use pty"
+    description:
+      "sudo can be configured to run only from a pseudo-pty"
+      "Note: visudo edits the sudoers file in a safe fashion, analogous to vipw(8). visudo locks the"
+      "sudoers file against multiple simultaneous edits, provides basic sanity checks, and checks for"
+      "parse errors. If the sudoers file is currently being edited you will receive a message to try"
+      "again later. The -f option allows you to tell visudo which file to edit."
+    rationale:
+      "Attackers can run a malicious program using sudo, which would again fork a background"
+      "process that remains even when the main program has finished executing."
+      "This can be mitigated by configuring sudo to run other commands only from a pseudo-pty,"
+      "whether 1/0 logging is turned on or not."
+    remediation:
+      "Edit the file /etc/sudoers orafile in /etc/sudoers.d/ with visudo or visudo"
+      "and add the following line:"
+      "Defaults use_pty"
+    cis_benchmark: {
+      profile_level: 1
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      "  check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{"
+      "         path:\"/etc/sudoers\""
+      "       }"
+      "      }"
+      "      files_to_check:{files_in_dir:{"
+      "         dir_path:\"/etc/sudoers.d/\""
+      "       }"
+      "      }"
+      "    content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"^\\\\s*Defaults\\\\s+([^#]\\\\S+,\\\\s*)?use_pty\b\""
+      "          expected_regex: \"Defaults use_pty\""
```

Comment:
Can you allow more whitespace in the expected_regex, like you do in other checks?